dev/com.ibm.ws.logging.hpel_fat/publish/servers/HpelServer2/bootstrap.properties

```diff
@@ -1,3 +1,4 @@
 bootstrap.include=../testports.properties
 websphere.log.provider=binaryLogging-1.0
 com.ibm.ws.logging.console.log.level=INFO
+com.ibm.ws.logging.trace.specification=com.ibm.ws.logging.*=all:com.ibm.ws.org.*=all=enabled
```

Comment:
How much trace is generated at the end of the test?

Reply:
The binary tracedata is 64KB.